public/app/features/plugins/sql/components/visual-query-builder/SelectRow.tsx

```diff
@@ -60,6 +69,26 @@ export function SelectRow({ sql, columns, onSqlChange, functions }: SelectRowPro
     [onSqlChange, sql]
   );
 
+  const onAliasChange = useCallback(
+    (item: QueryEditorFunctionExpression, index: number) => (alias: SelectableValue<string>) => {
+      let newItem = { ...item };
+
+      if (alias !== null) {
+        newItem = { ...item, alias: `"${alias?.value?.trim()}"` };
```

Comment:
Made changes here to trim whitespace and double quote. This seems like it could be a little ham fisted? Would be happy for feedback here.

Reply:
I think it's ok. The only issue I can see is if someone didn't know it was going to be double quoted and entered something in uppercase expecting it to show up in lowercase, as quoting it does preserve capitalization, but it's such a small issue I think we can just mention it in the docs

Reply:
Perhaps in the future we should add tooltips to the controls in the query editor, could describe the quoting behavior in there.

Reply:
That's a really good idea and I completely agree!